include/glow/LLVMIRCodeGen/LLVMIRGen.h

```diff
@@ -371,6 +371,8 @@ class LLVMIRGen {
   /// Generates LLVM IR that materializes the string literal \p str.
   virtual llvm::Value *emitStringConst(llvm::IRBuilder<> &builder,
                                        llvm::StringRef str);
+  /// Generate pointer to the host side file printer for JIT code to use. \p F.
```

Comment:
The comment looks a bit strange at the end. I mean `\p F`. BTW, how is F used in the function?